### ISSUE TITLE
Step 6 breaks with underscore in chromosome names.

### ISSUE THREAD

**Issue by aakashsur** [yezhengSTAT/mHiC#11]
It looks like step 6 breaks with underscores in chromosome names as well. 

It can be isolated to:

https://github.com/yezhengSTAT/mHiC/blob/0e5c13aa94349b8089a5ca48be8f5cbfbfc90981/mhic_step0-6.sh#L172

https://github.com/yezhengSTAT/mHiC/blob/0e5c13aa94349b8089a5ca48be8f5cbfbfc90981/s6_em_cython.pyx#L94

https://github.com/yezhengSTAT/mHiC/blob/0e5c13aa94349b8089a5ca48be8f5cbfbfc90981/s6_em_cython.pyx#L153

https://github.com/yezhengSTAT/mHiC/blob/0e5c13aa94349b8089a5ca48be8f5cbfbfc90981/s6_em_cython.pyx#L184

https://github.com/yezhengSTAT/mHiC/blob/0e5c13aa94349b8089a5ca48be8f5cbfbfc90981/s6_em_cython.pyx#L291

**Comment by yezhengSTAT** [comment]
Yes, again we assume the chromosome name has no underscore and is consistent for the whole processing procedure.
